packages/web/app/pages/api/proxy.ts

```diff
@@ -59,6 +59,10 @@ async function graphql(req: NextApiRequest, res: NextApiResponse) {
     captureException(error);
   }
 
+  if (!accessToken && req.headers['authorization']) {
+    accessToken = req.headers['authorization'].replace('Bearer ', '');
+  }
+
```

Comment:
I'm interested to know what and why here

Reply:
So I can point the CLI to this

Reply:
hmm but we use `/graphql` endpoint for GraphQL stuff

Reply:
this proxy is meant only for `@hive/app` to check the user, we built it because of auth0 limitations

Reply:
I wanted to avoid spinning up two proxies for testing the CLI integration from CI (GitHub actions).
I can remove it, though then I always will re-add it when working on testing the integrations :D